br/pkg/streamhelper/config/advancer_conf.go

```diff
@@ -15,7 +15,7 @@ const (
 	flagAdvancingByCache = "advancing-by-cache"
 
 	DefaultConsistencyCheckTick = 5
-	DefaultTryAdvanceThreshold  = 3 * time.Minute
+	DefaultTryAdvanceThreshold  = 108 * time.Second
```

Comment:
Why 108, not an integer ？

Reply:
I want to make it reduced by the same proportion.

flush time interval 5min -> TiKV flush interval 4 min -> TiDB try advance threshold 3 min
So if flush time interval is 3 min, the default try advance threshold is 3min*0.8*0.75=108s